src/Delta.proto

```diff
@@ -37,12 +39,26 @@ package Catalyst.Protocol.Delta;
  * CBEntries: one per active worker
  */
  message Delta {
-	uint32 version = 1; 
-	bytes PreviousDelta = 2;
+	uint32 Version = 1; 
+	bytes PreviousDeltaDfsHash = 2;
 	bytes MerkleRoot = 3;
 	bytes MerklePoda = 4;
 	uint32 TimeStamp = 5;
 	repeated Transaction.STTransactionEntry STEntries = 6;
 	repeated Transaction.CFTransactionEntry CFEntries = 7;
 	repeated Transaction.CoinbaseEntry CBEntries = 8;
 }
+
+/**
+ * CandidateDelta
+ *
+ * DeltaHash: The hash computed for the current delta produced by ProducerId.
+ *	This is meant to be used when voting for most popular hashes.
+ * PreviousDeltaDfsHash: DFDS address for the content of the delta preceding this candidate.
+ * ProducerId: Identifier of the producer of the candidate.
+ */
+message CandidateDelta {
+	bytes Hash = 1;
+	Common.PeerId ProducerId = 2;
+	bytes PreviousDeltaDfsHash = 3;
```

Comment:
Am I correct to assume that you will need this in order to compare CandidateDelta.Hash with Delta locally computed when implementing sharding? Is it useful at the moment? If no secondary chain, every node should have the same PreviousDeltaDfsHash right?

Reply:
I think it's to chain the deltas together.